.github/workflows/publish-idf-rust-tags.yml

```
@@ -6,7 +6,7 @@ on:
       toolchain-version:
         description: "Version of Rust toolchain"
         required: true
-        default: "1.80.0.0"
+        default: "v1.82.0.2"
```

I think we dont need the `v`
```suggestion
        default: "1.82.0.2"
```

---

In this case, the `toolchain-version` argument is only used for the name of the tag, and all our previous tags dont have the `v`. We can start adding it, but people would be confused. 

https://hub.docker.com/r/espressif/idf-rust/tags

---

I can revert it for the docker action, but I think we need it for the rest - does that work for you?

---

Yes, we only need it without `v` in publish-idf-rust-tags.yml, the rest require it.

---

I applied this change to #278